.travis.yml

```diff
@@ -114,10 +116,10 @@ matrix:
     - env: B=style-check
       dist: trusty
       sudo: required
-      before_script:
-      - astyle --version
+      before_install:
+      - docker pull balabit/syslog-ng-image-bionic
       script:
-      - scripts/style-checker.sh format
+      - ./dbld/rules make-style-format
```

Comment:
Running the same script inside dbld image would be fine - even better than calling any configure & make. I could imagine something like this: `dbld/rules shell scripts/style-checker.sh format` (without knowing much about dbld).